webpack/src/utils.ts

```diff
@@ -105,7 +106,10 @@ export function getWebpackConfig () {
       resolve: {
         extensions: ['.json', '.js', '.jsx', '.css']
       },
-      devtool: 'source-map'
+      devtool: 'source-map',
+      devServer: {
+        publicPath: path.join('/${bundlePath}/')
```

Comment:
I had tried it in the past with just `publicPath: 'dist'` and not `publicPath: '/dist/'`, but webpack-dev-server would not serve the bundle in the browser.

Burnt about 1-2 hours of debugging time trying to figure this one out.